### DISCUSSION TITLE
向量化引擎NULL值问题

### DISCUSSION THREAD

**Discussion by yaoyi** [apache/doris#12626] [A - General / Q&A]
请问下面这个查询，除了关掉向量化引擎外，有其他途径可以获取正常的返回吗, 向量化引擎这里没办法判断NULL值，下面这个查询是需要获取没有打标签的产品列表
```sql
SELECT
  products.id 
FROM
  `products`
LEFT JOIN taggings ON taggings.account_id = products.account_id 
  AND taggings.marketplace_id = products.marketplace_id 
  AND taggings.sku = products.sku
WHERE
  `products`.`account_id` = 981
  AND `taggings`.`id` IS NULL
```

**Comment by JNSimba** [comment]
what is the doris version？

**Reply by yaoyi** [reply]
1.1.2

**Reply by JNSimba** [reply]
Can you provide the schema and some data of the table, we try to simulate it

**Comment by zy-kkk** [comment]
Hello, I don't know if your query column(`AND `taggings`.`id` IS NULL`) is bloom filter column, we have recently fixed a related problem, you can try to download the latest 1.1.3 for testing
PR:#13299
